src/richText.js

```diff
@@ -103,7 +102,18 @@ define([
         }
     });
 
-    CKEDITOR.config.allowedContent = true;
+    CKEDITOR.config.allowedContent = {
+        span: {
+            classes: 'label,label-*',
+            attributes: 'data-output-value,data-value,data-date-format',
+        },
+        i: {
+            classes: 'fcc,fcc-*,fa,fa-*',
+        },
+        output: {
+            attributes: 'value',
+        },
```

Comment:
I'm not sure why markdown would matter here? Within ckeditor you only use plain text like `*bold*` or `**italics**` it's only rendered in html in a separate div using markdown-it